AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -2570,6 +2570,14 @@ public void showContextMenuDeckOptions() {
         }
     }
 
+    //Callback to open card browser for currently selected deck
```

Comment:
Could you use Java Doc Comment syntax so this is picked up by the IDE?

 ```suggestion
    /** Callback to open card browser for currently selected deck */
```